community/xxhash/APKBUILD

```diff
@@ -67,10 +58,14 @@ _py() {
 	depends=xxhash
 	install_if="py-xxhash=$pkgver-r$pkgrel $python"
 
-	cd "$pybuilddir"
+	cd "$_pybuilddir"
 	$python setup.py install --prefix=/usr --root="$subpkgdir"
 }
 
-sha512sums="085643b52e091ac0eedd54c4459220b3643d825ca71a11e952d00ea2041c570ff57d8553d0378f34e038ca9ee3b40d2048ed02d44d5aff1fbfcbf5e642487ba0  xxHash-0.6.5.tar.gz
-72a99d744ccaac830e9789053acb9728b2da457c7841e2aae96e9748450f09366b9830f6d92b62ac494e938f43c1fea7910c9d5257824ae33c1fe48f199ed9cc  python-xxhash-0.6.5.tar.gz
+libs() {
```

Comment:
This seems [redundant](https://git.alpinelinux.org/abuild/tree/abuild.in#n1783).